com.unity.render-pipelines.core/Runtime/Debugging/Prefabs/Scripts/DebugUIHandlerObjectList.cs

```diff
@@ -1,69 +1,20 @@
-using UnityEngine.UI;
-
 namespace UnityEngine.Rendering.UI
 {
     /// <summary>
     /// DebugUIHandler for object list widget.
     /// </summary>
-    public class DebugUIHandlerObjectList : DebugUIHandlerWidget
+    public class DebugUIHandlerObjectList : DebugUIHandlerField<DebugUI.ObjectListField>
```

Comment:
Thanks for your TED talk, really cool :). This was implemented by @adrien-de-tocqueville, if I am not wrong, maybe we could rethink about this display for runtime.

Reply:
Very low priority though, since it's only used by 1 thing right now. I'd say let's keep this in mind and when/if we do UIToolkit refactor, implement it differently at that point.

Reply:
Cool, lets hope no many users use this :)

Reply:
yeah honestly i just put it that way because it was easy to do and it's for debug anyway
But feel free to make it a real list if you want 😄